Makefile

```diff
@@ -1,11 +1,12 @@
+# Copyright 2019 Intel Corporation
 # Copyright IBM Corp. All Rights Reserved.
 #
 # SPDX-License-Identifier: Apache-2.0
 
 TOP = .
 include $(TOP)/build.mk
 
-SUB_DIRS = utils ercc ecc_enclave ecc tlcc_enclave tlcc integration # docs
```

Comment:
I actually left the docs there commented out on purpose to explicitly show that we do not make docs now.  But admittedly not an essential thing ...

Reply:
oh, did I remove that?! .. restored!